applications/asset_tracker/README.rst

```diff
@@ -52,19 +57,22 @@ Requirements
 User interface
 **************
 
-The two buttons and two switches are used to enter a pairing pattern to associate a specific board with an nRF Cloud user account.
-In addition, the two switches have the following function when the connection is established:
+The buttons and switches have the following functions when the connection is established:
 
-Switch 1:
-    * Toggle to simulate flipping of the boards orientation.
+Button 1:
+    * Send a BUTTON event to nRF Cloud.
+    * Toggle GPS operation (long press the button for a minimum of 10 seconds).
```

Comment:
Yes, that is valid for both boards. The intention is that all text is valid for all boards, unless otherwise noted.

The dictionary definition of toggle (v.) was more vague than I thought. Edited to be more precise.

Simulated GPS operation is not enabled by default on any board, and is not mentioned in the documentation.